### DISCUSSION TITLE
Save a scene

### DISCUSSION THREAD

**Discussion by LighthouseMedia** [home-assistant/frontend#18998] [Accessibility]
Have a scene.save command, where entities are updated to current values. This would allow non-tech savy individules to make and save scenes, without going to the Settings. This would also allow it to be used in automations, for example to save the state of a house when someone leaves then turn off everything, but then they return call the scene and have it back the way it was when they left.

**Comment by karwosts** [comment]
This already exists, it is called `scene.create`.

https://www.home-assistant.io/integrations/scene/#creating-scenes-on-the-fly

**Reply by LighthouseMedia** [reply]
Scene.create does not store them after a restart, nor does it allow you to eaisly update the values. What im sugesting is more like a scene.update.

**Comment by Fartukanoja** [comment]
> Have a scene.save command, where entities are updated to current values. This would allow non-tech savy individules to make and save scenes, without going to the Settings. This would also allow it to be used in automations, for example to save the state of a house when someone leaves then turn off everything, but then they return call the scene and have it back the way it was when they left.
